WORKSPACE

```diff
@@ -0,0 +1,43 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+##########
+# Protobuf
+##########
+http_archive(
+    name = "rules_proto",
+    sha256 = "66bfdf8782796239d3875d37e7de19b1d94301e8972b3cbd2446b332429b4df1",
+    strip_prefix = "rules_proto-4.0.0",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/refs/tags/4.0.0.tar.gz",
+        "https://github.com/bazelbuild/rules_proto/archive/refs/tags/4.0.0.tar.gz",
+    ],
+)
```

Comment:
I was looking at the `rules_proto` repo and this release seems to be from Sep 15, 2021. There have been newer releases since then, such as one on Apr 05, 2022 (however, that has a version number '4.0.0-3.20.0' which I don't know how to interpret). Is there a specific reason for picking the older release?

Reply:
No specific reason. We only use basic features so I suspect it doesn't matter what version we use. Either way, I updated to the latest version and added a comment explaining how to keep the version updated.